### PR TITLE
Add some more runtime type support.

### PR DESCRIPTION
**Pull request by brson**
This makes u256 work in particular.

### CODE REVIEW

File: language/tools/move-mv-llvm-compiler/tests/rbpf-tests/debug-print.move [hunk 1]
```diff
@@ -1,13 +1,71 @@
 // log 10
+// log 11
+// log 12
+// log 13
+// log 14
+// log 15
+// log @00000000000000000000000000000000000000000000000000000000000000AB
+// log true
```
Comment (jcivlin) [inline language/tools/move-mv-llvm-compiler/tests/rbpf-tests/debug-print.move:8]: I wonder what these lines (esp. long line at 7) stand for? Can you explain, please.

**Review by jcivlin — APPROVED** [review]
Code looks good

File: language/tools/move-mv-llvm-compiler/tests/rbpf-tests/debug-print.move (hunk 1 above)
Comment (nvjle) [reply language/tools/move-mv-llvm-compiler/tests/rbpf-tests/debug-print.move:8]: > I wonder what these lines (esp. long line at 7) stand for? Can you explain, please.

See `move-mv-llvm-compiler/docs/Development.md` which explains `log` and the other directives.

**Comment by nvjle** [comment]
Another LGTM from me as well, thanks.

File: language/tools/move-mv-llvm-compiler/tests/rbpf-tests/debug-print.move (hunk 1 above)
Comment (nvjle) [reply language/tools/move-mv-llvm-compiler/tests/rbpf-tests/debug-print.move:8]: Also, the long hex number starting with `@` on line 7 is a Move address. See `https://move-language.github.io/move/address.html#address`. FWIW, I recommend reading the whole Move book.

Comment (jcivlin) [reply language/tools/move-mv-llvm-compiler/tests/rbpf-tests/debug-print.move:8]: Ty, Jason.

**Pull request merged**